pyramid_mailer/mailer.py

```diff
@@ -16,6 +16,20 @@
 from pyramid_mailer._compat import SMTP_SSL
 
 
+def _check_bind_options(kw):
```

Comment:
Maybe.  Right now `Mailer.bind` does its own equivalent checks, so the call would be redundant.  (Those checks are intertwingled with the extraction of the pertinent parameters.)

Another way to go, which I considered initially, would be to have a common `_extract_bind_options` that gets called by all three `.bind` methods which does both the check and the extraction of the `default_sender` and `transaction_manager` options.  (The extracted parameters would then be ignored by everyone except `Mailer.bind`.)  The ugliness with doing that has to do with the way the default values for these options is determined: in the real `Mailer`, defaults for these come from object properties; in the dummy/debug mailers, these parameters, ignored as they are, have indeterminate (or inconsequential) default values. (If that makes any sense...)